docs/faq.md

```diff
@@ -105,53 +127,91 @@ salt-call -c /opt/watchmaker/salt state.show_top
+directly-use the underlying SaltStack functionality to effect the desired
+change. This is done from the commandline - as root - by executing:
 
 *   Disable FIPS-mode: `salt-call -c /opt/watchmaker/salt ash.fips_disable`
 *   Enable FIPS-mode: `salt-call -c /opt/watchmaker/salt ash.fips_enable`
 
 And then rebooting the system.
 
-## How do I install watchmaker when I am using Python 2.6?
-
-While Watchmaker no longer offically supports Python 2.6, you may use the last
-version where it was tested, Watchmaker 0.21.7. That version includes pins on
-dependencies that will work for Python 2.6.
-
-However, there are three python "setup" packages needed just to install ``watchmaker``,
-and these packages cannot be platform-restricted within the ``watchmaker`` package
-specification.
-
-Below is the list of packages in question, and the versions that no longer
-support Python 2.6:
-
-*   ``pip>=10``
-*   ``wheel>=0.30.0``
-*   ``setuptools>=37``
-
-In order to install pip in Python 2.6, you can get it from:
-
-*   <https://bootstrap.pypa.io/pip/2.6/get-pip.py>
-
-Once a Python 2.6-compatible ``pip`` version is installed, you can install
-compatible versions of the other packages like this:
-
-```shell
-python -m pip install --upgrade "pip<10" "wheel<0.30.0" "setuptools<37"
-```
-
-You can then [install watchmaker](installation) by restricting the watchmaker
-version to the last version tested with Python 2.6:
-
-```shell
-python -m pip install "watchmaker==0.21.7"
-```
-
 ## How do I get Watchmaker release/project notifications?
 
-Users may use an RSS reader of their choice to subscribe to the Watchmaker Release feed to get notifications on Watchmaker releases. The Watchmaker RSS release feed is https://github.com/plus3it/watchmaker/releases.atom.
-
-Users can also "watch" the GitHub project to receive notifications on all project activity, https://github.com/plus3it/watchmaker/subscription.
+Users may use an RSS reader of their choice to subscribe to the Watchmaker
+Release feed to get notifications on Watchmaker releases. The Watchmaker RSS
+release feed is https://github.com/plus3it/watchmaker/releases.atom.
+
+Users can also "watch" the GitHub project to receive notifications on all
+project activity, https://github.com/plus3it/watchmaker/subscription.
+
+## Why do I get warnings in my system logs about the rsyslog service not being able to connect to the `logcollector` host
+
+Watchmaker leverages the `oscap` utility and associated OSCAP content for a
+significant percentage of its hardening-actions. Recent updates to this content
+have included (inappropriately) adding:
+
+~~~
+# Per CCE-80863-4: Set *.* @@logcollector in /etc/rsyslog.conf
+*.* @@logcollector
+~~~
+
+To the `/etc/rsyslog.conf` file. The `*.* @@logcollector` line results in the
+`rsyslog` service attempting to do remote-logging to the external host
+`logcollector`. If no such hostname exists in the hardened-system's DNS domain,
```

Comment:
Woah. Is this reported to the `ComplianceAsCode/content` project? All someone needs to do to get logs from any host it runs against would be to create this one DNS entry???

Reply:
Not yet. Was working on the EL8 `/boot` issues. I'll go report it once I've finished processing the current comments on this PR.

Reply:
[Reported](https://github.com/ComplianceAsCode/content/issues/11812)